string node_name
uint8 transition
---
bool success
